tests/main/security-private-tmp/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Ensure that the security rules for private tmp are in place.
+
+prepare: |
+    echo "Given a basic snap is installed"
+    snapbuild $TESTSLIB/snaps/test-snapd-tools .
+    snap install test-snapd-tools_1.0_all.snap
+
+    echo "And another basic snap is installed"
+    SNAP_INSTALL_DIR=$(mktemp -d)
+    cp -ra $TESTSLIB/snaps/test-snapd-tools/* $SNAP_INSTALL_DIR
```

Comment:
Quotes around $SNAP_INSTALL_DIR (there and following) is slightly nicer as this would break once we have spaces in filenames. In practise it does not make any difference because mkdtemp -d never has spaces of course.